kd/deployment.yaml

```
@@ -0,0 +1,255 @@
+          - name: CLIENT_MAX_BODY_SIZE
+            value: '52'  
+        volumeMounts:
+          - name: certs
+            mountPath: /certs
+            readOnly: true
+        ports:
+          - name: https
+            containerPort: 10443
+        resources:
+          limits:
+            memory: 32Mi
+            cpu: 400m
+          requests:
+            memory: 4Mi
+            cpu: 100m
+
+      - name: hocs-camunda-tools
+        image: quay.io/ukhomeofficedigital/hocs-camunda-tools:0.0.4
+        imagePullPolicy: Always
+        securityContext:
+          runAsNonRoot: true
+          capabilities:
+            drop:
+             - SETUID
+             - SETGID
+        env:
+          - name: DB_HOST
+            value: 'postgres'
+          - name: HOCS_BASICAUTH
+            valueFrom:
+              secretKeyRef:
+                name: ui-casework-creds
+                key: plaintext
+          - name: DB_HOST
+            valueFrom:
+              secretKeyRef:
+                name: {{.KUBE_NAMESPACE}}-rds
+                key: endpoint
+          - name: DB_PORT
+            valueFrom:
+              secretKeyRef:
+                name: {{.KUBE_NAMESPACE}}-rds
+                key: port
+          - name: DB_NAME
+            valueFrom:
+              secretKeyRef:
+                name: {{.KUBE_NAMESPACE}}-rds
+                key: db_name
+          - name: DB_SCHEMA_NAME
+            valueFrom:
+              secretKeyRef:
+                name: hocs-workflow
+                key: rds_schema
+          - name: DB_USERNAME
+            valueFrom:
+              secretKeyRef:
+                name: hocs-workflow
+                key: rds_user
+          - name: DB_PASSWORD
+            valueFrom:
+              secretKeyRef:
+                name: hocs-workflow
+                key: rds_password
+        resources:
+          limits:
+            cpu: 900m
+            memory: 512Mi
+          requests:
+            cpu: 200m
+            memory: 80Mi
+        ports:
+          - name: http
+            containerPort: 8080
+        livenessProbe:
+          httpGet:
+            path: /actuator/health
+            port: http
+            httpHeaders:
+              - name: X-probe
+                value: kubelet
+          initialDelaySeconds: 45
+          periodSeconds: 20
+          timeoutSeconds: 10
+        readinessProbe:
+          httpGet:
+            path: /actuator/health
+            port: http
+            httpHeaders:
+              - name: X-probe
+                value: kubelet
+          initialDelaySeconds: 45
+          periodSeconds: 20
+          timeoutSeconds: 10
+        volumeMounts:
+          - mountPath: /certs
+            name: certs
+            readOnly: true
+          - mountPath: /etc/ssl/certs
+            name: bundle
+            readOnly: true
+      
+      - name: keycloak-proxy-web
```

As there's only one:
```suggestion
      - name: keycloak-proxy
```